inginious/frontend/common/static/css/bootstrap-modal-fullscreen.css

```diff
@@ -0,0 +1,46 @@
+/* Copy-paste from http://www.minimit.com/demos/bootstrap-3-transparent-and-fullscreen-modals */
```

Comment:
It is written in a lot of places on the website but I asked confirmation

![selection_052](https://cloud.githubusercontent.com/assets/7595324/25224185/e9f19522-25bd-11e7-9ce8-9a7a2359129f.png)